src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/UserProfileController.java

```diff
@@ -202,24 +209,28 @@
         return ResponseEntity.ok(response);
     }
 
-    @ApiOperation(value = "Retrieves user profile queried by email or userId. If both provided email is preferred", 
-                  authorizations = {
+    @ApiOperation(value = "Retrieves a User Profile by Email or ID. If both are present then Email is used to retrieve.",
+            authorizations = {
                     @Authorization(value = "ServiceAuthorization"),
                     @Authorization(value = "Authorization")
-                  })
+            })
     @ApiResponses({
             @ApiResponse(
                     code = 200,
-                    message = "Representation of a user profile data",
+                    message = "Representation of a User profile",
                     response = UserProfileResponse.class
             ),
             @ApiResponse(
                     code = 400,
-                    message = "Bad Request"
+                    message = "An invalid request has been provided"
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = FORBIDDEN_ERROR_ACCESS_DENIED
             ),
             @ApiResponse(
                     code = 404,
-                    message = "Not Found"
+                    message = "No User Profile found with the given ID"
             ),
             @ApiResponse(
                     code = 500,
```

Comment:
add the message = "Internal Server Error"

Reply:
It has been added, GitHub has collapsed the unchanged lines, please expand and you will see it on line 237 :)